### DISCUSSION TITLE
Login Credentials Missing

### DISCUSSION THREAD

**Discussion by mikesch13** [tgorgdotcom/locast2plex#294] [Q&A]
No matter what I have tried I keep getting "Loading Configuration File: /app/config/config.ini
locast2plex_1  | Locast Login Credentials Missing. Exiting..."

The config file is present and has this in it: (usernanme and pass redacted) 
[main]
#uuid=<note: automaticially created if nonexistant>
plex_accessible_ip = 192.168.0.11
locast_username=ms*******@mac.com
locast_password=**********

#disable_ssdp = False
#plex_accessible_port = 6077
#use_old_plex_interface = False
#tuner_count = 3
#concurrent_listeners = 10
#cache_dir =
#ffmpeg_path = None
#ffmpeg_buffer = 9123840
#verbose = False
#bytes_per_read=1152000
#tuner_type = Antenna
#fcc_delay = 43200
#epg_update_frequency = 43200
#epg_update_days = 7
#reporting_model=xxxxxx
#reporting_firmware_name=xxxxxx
#reporting_firmware_ver=xxxxxx

Any ideas why Docker Keeps erroring out like this? 

![Screen Shot 2021-08-29 at 12 02 48 PM](https://user-images.githubusercontent.com/89745317/131260750-8c244267-6072-4040-9396-caef4cf36a87.png)

I noticed this on GitHub page but don't know how to do it: move config.ini into /app/config/

**Comment by wgstarks** [comment]
Not sure if this will fix your issue but my .ini file has spaces before and after the equal sign for user and password.
